i8_terminal/types/i8_completer.py

```diff
@@ -45,7 +45,21 @@ def get_completions(self, document: Document, complete_event: CompleteEvent = No
         filter_choices = True
 
         if ctx.last_option:
-            matched_params = [p for p in command.params if isinstance(p, click.Option) and ctx.last_option in p.opts]
+            if not document.is_cursor_at_the_end:
```

Comment:
The following is duplicate of the above! Move duplicate logic to a function or method.